safekeeper/spec/MCProposerAcceptorReconfig.tla

```diff
@@ -0,0 +1,41 @@
+---- MODULE MCProposerAcceptorReconfig ----
+EXTENDS TLC, ProposerAcceptorReconfig
+
+\* Augments the spec with model checking constraints.
+
+\* It slightly duplicates MCProposerAcceptorStatic, but we can't EXTENDS it
+\* because it EXTENDS ProposerAcceptorStatic in turn. The duplication isn't big
+\* anyway.
```

Comment:
Thanks for this comment, but, more useful for future maintenance would be a comment both here and in `MCProposerAcceptorStatic` would be to remind future readers/changers to see whether the change needs to be duplicated in the other file.

Reply:
done